### PR TITLE
[android] Added milliseconds to log record

### PR DESCRIPTION
**Pull request by alexzatsepin**
https://jira.mail.ru/browse/MAPSME-3831
@mpimenov PTAL

### CODE REVIEW

File: android/src/com/mapswithme/util/log/FileLoggerStrategy.java [hunk 1]
```diff
@@ -18,6 +18,7 @@
 import java.io.FileWriter;
 import java.io.IOException;
 import java.text.DateFormat;
+import java.text.SimpleDateFormat;
```
Comment (mpimenov) [inline android/src/com/mapswithme/util/log/FileLoggerStrategy.java]: Sort imports?

**Comment by mpimenov** [comment]
Why do we need such precision?

File: android/src/com/mapswithme/util/log/FileLoggerStrategy.java (hunk 1 above)
Comment (alexzatsepin) [inline android/src/com/mapswithme/util/log/FileLoggerStrategy.java]: Imports are managed by IDE, manual sorting is no needed. And it looks sorted, doesn't it?

**Comment by alexzatsepin** [comment]
This precision is needed to make our QA able to test the parameter 'Geo location request interval' which depends on my position mode. For example, in car routing that parameter is 500 ms.

File: android/src/com/mapswithme/util/log/FileLoggerStrategy.java (hunk 1 above)
Comment (mpimenov) [inline android/src/com/mapswithme/util/log/FileLoggerStrategy.java]: Oh, I missed it's text*, and the next is util*.

**Pull request merged**